testsuite/features/core_ubuntu_salt.feature

```diff
@@ -17,33 +23,40 @@ Feature: Be able to bootstrap an Ubuntu minion and do some basic operations on i
     And I enter "root" as "user"
     And I enter "linux" as "password"
     And I select "1-UBUNTU-TEST" from "activationKeys"
-    And I select the hostname of the proxy from "proxies"
     And I click on "Bootstrap"
-    And I wait until I see "Successfully bootstrapped host! " text
+    Then I wait until I see "Successfully bootstrapped host!" text
```

Comment:
And

Reply:
I don't agree that one... we'll have the whole scenario without Then, just because we understand that step like and action, when is clearly expecting something (main reason for a ´Then´).
But for the sake of that PR, I will just change.

Reply:
If the "wait until I see" looks so bad for you... I suggest we hide that inside the step and we use "I should see a" with a long timeout. I guess in that way, we can have here a `Then`....

Reply:
OK @srbarrios.

But then the step should be renamed to something with "should", so the things that is expected becomes clear. For example:
```
  Then "Successfully bootstrapped host!" should appear after some time
```

We could have that step under two names, one for which we are more interested in the action (= the main point is to wait), and the other one for which we are more interested in the test (= the main point is to see some text).

Would that be acceptable?